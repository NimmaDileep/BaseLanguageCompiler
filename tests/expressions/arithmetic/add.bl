// Addition currently not working

var x: Int = 5;
var y: Int = 1;

print(x + y);

y = -2;

print(x + y);

var p: Float = 1.5;
var q: Float = 2.7;
print(p + q);
print(x + q);

