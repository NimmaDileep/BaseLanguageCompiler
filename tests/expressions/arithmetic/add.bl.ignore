// Addition currently not working

var x: Int = 5;
var y: Int = 1;

print(x + y);

y = -2;

print(x + y);
