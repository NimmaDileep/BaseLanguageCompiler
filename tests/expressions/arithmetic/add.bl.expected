6
3
4.2
7.7
-2.6
2.6